config/redis.yml

```diff
@@ -4,6 +4,10 @@ development:
 test:
   host: localhost
   port: 6379
+staging:
+  host: ENV['REDIS_URL']
```

Comment:
Great. thanks! I'd say we use the env vars whenever possible? I'll modify the file accordingly.